apis/cache/v1beta1/cloudmemorystore_instance_types.go

```diff
@@ -38,6 +38,14 @@ type CloudMemorystoreInstanceParameters struct {
 	// +immutable
 	Tier string `json:"tier"`
 
+	// TransitEncryptionMode specifies TLS encryption mode for Redis service.
+	// For TRANSIT_ENCRYPTION_MODE_UNSPECIFIED the TLS in transit encryption is not set.
+	// If set to SERVER_AUTHENTICATION the client-to-server traffic encryption is enabled.
+	// Encryption is disabled if the value is DISABLED
+	// https://cloud.google.com/memorystore/docs/redis/about-in-transit-encryption
+	// +optional
+	TransitEncryptionMode string `json:"transitEncryptionMode,omitempty"`
```

Comment:
We have a guideline to use pointers for optional fields in the `Parameters` struct. So this field should be:

```suggestion
	// +optional
	TransitEncryptionMode *string `json:"transitEncryptionMode,omitempty"`
```